src/ClimateDynamicsComponent.jl

```diff
@@ -7,36 +7,33 @@
     # Average global temperature
     temp = Variable(index=[time])
 
-    # LifeTempConst
-    LifeTempConst = Parameter()
+    # lifetempconst
+    lifetempconst = Parameter(default = -42.7421952811)
 
-    # LifeTempLin
-    LifeTempLin = Parameter()
+    # lifetemplin
+    lifetemplin = Parameter(default = 29.0603120788)
 
-    # LifeTempQd
-    LifeTempQd = Parameter()
+    # lifetempqd
+    lifetempqd = Parameter(default = 0.0014564222822)
 
     # Climate sensitivity
-    ClimateSensitivity = Parameter()
-end
-
-function run_timestep(s::climatedynamics, t::Int)
-    v = s.Variables
-    p = s.Parameters
-    d = s.Dimensions
+    climatesensitivity = Parameter(default = 2.999999803762826)
```

Comment:
Lets make that 3.0

Reply:
Doing this causes our integration tests that compare to old results to fail since we have such tight bounds on the precision for the tests.  I can loosen that precision, but for now let's leave this until the end so we can make sure our moving around of default parameters keeps things consistent.